xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-resource/resourcePicker.css

```diff
@@ -19,6 +19,18 @@
  */
 #template('colorThemeInit.vm')
 
+/* We need to redefine this class to have better priority than the CKEditor CSS reset. */
```

Comment:
I'd be interested to have an explanation of why this class is needed.
What happens when this CSS is not defined?

Reply:
![image](https://github.com/user-attachments/assets/608fef22-8fa3-4d77-82af-88d5f02d3f3a)

If this CSS is not defined, we cannot use the XWiki defined class `.sr-only` to hide visually elements that are only useful for screen readers.
This is an issue because it's a class we should be able to rely on in all our codebase.

The CKEditor reset takes precedence over our definition of `.sr-only` and decides the style of this element:
![image](https://github.com/user-attachments/assets/7f2fdf59-f732-4e2d-b7c5-e01cd4ac4c99)

Reply:
Updated the comment to provide more detail in https://github.com/xwiki/xwiki-platform/pull/3612/commits/5c7dfef433cb5fb7d4a33544f98a7d9413106913 :+1: